Startup spotlight: Unified.to (#6144)
PostHog/posthog.com: contents/blog/startup-unified.md

---
title: "Startup Spotlight: Unified.to"

> date: 2023-06-19

Author: ```suggestion
date: 2023-06-21
```

author: ["joe-martin"]
showTitle: true
rootpage: /blog
sidebar: Blog
hideAnchor: true
featuredImage: ../images/blog/spotlight_unified.png
featuredImageType: full
category: Startups
---

> This month we’re chatting to [Unified.to](https://unified.to/) co-founder and CEO, Roy Pereira. Roy is a serial entrepreneur, and Unified.to is his fifth startup. Previously, Roy had been a GM at [Vendasta](/customers/vendasta), the world’s leading end-to-end platform for channel partners selling digital solutions to small and medium-sized businesses.

Reviewer: Previously, Roy had been the CEO and founder of CalendarHero (aka. Zoom.ai), which was acquired in 2021 by Vendasta.

Author: ```suggestion
This month we’re chatting to [Unified.to](https://unified.to/) co-founder and CEO, Roy Pereira. Roy is a serial entrepreneur, and Unified.to is his fifth startup. Previously, Roy had been the CEO and founder of CalendarHero (formerly known as Zoom.ai), which was acquired by [Vendasta](/customers/vendasta) in 2021.
```

> Every month we shine the spotlight on a different startup to show off the cool things they're building, as part of [the PostHog for Startups program](/startups). Sign up today for $50k in PostHog credit and a bunch of other sweet perks!

### Hi Roy! Can you start by telling us a bit about Unified.to?
“Sure thing! Unified.to is a unified API company. We provide integrations-as-a-service to SaaS software companies. It’s the sort of product I really wish I had when I was building my previous startups, because building integrations to get customer data is very costly and time consuming.

We don’t believe software companies should be building customer-facing integrations anymore. It just takes up too many resources. We don’t host our websites on servers in a closet anymore, so why are we still building integrations? Businesses are starting to catch on to this. You see a lot of other X-as-a-service developer tools out there: Stripe for payments, Twilio for SMS, Okta for authentication. We believe integrations should also follow that model.”

> ### Interesting! How has the journey been so far?

Reviewer 2: ```suggestion
### How has the journey been so far?
```

Reviewer 2: This just sounds a bit weird, tonally.

> “Unified.to is five months old and, while you’d think we’re pretty early on, we’re actually not. We have [product-market fit](/blog/how-to-product-market-fit) and we’re beating incumbent competitors when we face off against them, we’re getting our processes in order and our first customers. It’s quite exciting.

Reviewer 2: ```suggestion
“Unified.to is five months old and, while you’d think we’re pretty early on, we’re actually not. We have [product-market fit](/blog/product-market-fit-game) and we’re beating incumbent competitors when we face off against them, we’re getting our processes in order and our first customers. It’s quite exciting.
```

That was super important because it allowed our customers a lot of flexibility in terms of how often they synchronize customers' data, and is basically a one-to-one correspondence with how much they charge _their_ customers. If they're charging their customer a lot, then real-time synchronization is fine. If you're on a free plan or a very inexpensive plan, you probably don't want to do that. You'll probably want to do a daily synchronization, so that was really key.

The second thing we learned from our user interviews is that there was a specific need from HR software companies. We initially thought we’d focus on sales-oriented software companies, but nearly all the customers coming in were HR software teams. So, we tweaked the roadmap to reflect their interest.”

### Looking forward, what are you most excited about?

> “I'm a big data nerd and I geek out most on seeing what users are doing on my platform. I insist our teams don’t build anything they can’t measure, and that’s where PostHog comes in. I’m really excited to see the funnels, customer journeys, and understand how they change as we built. Then you can do a bunch of analysis and understand how your product _is_ being used instead of _assuming_ how it’s used.

Reviewer: as we grow.

Author: ```suggestion
“I'm a big data nerd and I geek out most on seeing what users are doing on my platform. I insist our teams don’t build anything they can’t measure, and that’s where PostHog comes in. I’m really excited to see the funnels, customer journeys, and understand how they change as we grow. Then you can do a bunch of analysis and understand how your product _is_ being used instead of _assuming_ how it’s used.
```

This sort of data makes your product better. It allows us to tweak our messaging as well, tweak the product, and generally make a lot of improvements. 

I'm also an API nerd — and this industry is all about APIs. The engineers that I work with, the marketers, everybody loves APIs because we're dealing with them all the time. I don't feel like APIs are going away. They're not shrinking, they're just expanding. Everybody has APIs. I think that's quite exciting.

### On the other hand, what are you most worried about?
“Well, I’m _not_ worried about AI! My last startup started off as an AI chatbot eight years ago, in the last hype cycle of AI chat. I saw people asking me when the AI robots were going to kill us and all that, and that was eight years ago. We're getting the exact same thing now. AI now is a thousand times more powerful, but it’s still not general AI.

> What I _do_ worry about is funding. We're going through a really weird and unique cycle in tech funding right now, of the type not seen since the dot com crash in the 2000s. Smart investors are putting money elsewhere, thanks to Covid, so that's causing a ripple effect throughout the entire market.

Reviewer: ...thanks to high interest rates...

Author: ```suggestion
What I _do_ worry about is funding. We're going through a really weird and unique cycle in tech funding right now, of the type not seen since the dot com crash in the 2000s. Smart investors are putting money elsewhere, thanks to high interest rates, so that's causing a ripple effect throughout the entire market.
```

> How long are we going to stay in this weird funding cycle? I'm hoping it is over by the end of this year. I don't think it's going to go back to the way it used to be, but I hope it improves.”

Reviewer: ...it used to be during the low-interest days of COVID stimulus funds, but...

Author: ```suggestion
How long are we going to stay in this weird funding cycle? I'm hoping it is over by the end of this year. I don't think it's going to go back to the way it used to be during the low-interest days of Covid stimulus funds, but I hope it improves.”
```

### Finally, as a serial entrepreneur, what advice would you give to other startups now?

> “I think a lot of tech entrepreneurs, like me, are very technical and I one issue there is that we overbuild before we launch. You can launch anytime you want, but it doesn't matter if a tree falls in the forest if nobody's there. 

Reviewer 2: ```suggestion
“I think a lot of tech entrepreneurs, like me, are very technical and one issue there is that we overbuild before we launch. You can launch anytime you want, but it doesn't matter if a tree falls in the forest if nobody's there. 
```
